compiler-rt/test/rtsan/test_rtsan.cpp

```diff
@@ -0,0 +1,17 @@
+// RUN: %clangxx -fsanitize=realtime %s -o %t
```

Comment:
Did MaskRay ask us for lit tests that check the mutual exclusivity of rtsan with the other sanitizers? Do you think this PR would be a good place to add those?

Reply:
Check these tests out in clang/test/Driver

Reply:
Nice - got it